pointmatcher/ErrorMinimizers/PointToPlane.cpp

```diff
@@ -203,14 +256,29 @@ typename PointMatcher<T>::TransformationParameters PointToPlaneErrorMinimizer<T>
 				 * Eigen::AngleAxis<T>(x(1), Eigen::Matrix<T,1,3>::UnitY())
 				 * Eigen::AngleAxis<T>(x(2), Eigen::Matrix<T,1,3>::UnitZ());*/
 
-				transform = Eigen::AngleAxis<T>(x.head(3).norm(),x.head(3).normalized());
+				if (!force4DOF)
+				{
+					transform = Eigen::AngleAxis<T>(x.head(3).norm(), x.head(3).normalized()); //x=[alpha,beta,gamma,x,y,z]
```

Comment:
Related to the above comments, this block hints at Gamma = yaw.

Can this notation be defined before?

Reply:
The intention was that Gamma would be the matrix and gamma would be the error rotation vector z component (sort of delta yaw). I guess I'll call the previous matrixGamma and this one stay gamma.